guides/micronaut-graphql-todo/kotlin/src/main/kotlin/example/micronaut/ToDoRepository.kt

```diff
@@ -0,0 +1,8 @@
+package example.micronaut
+
+import io.micronaut.data.jdbc.annotation.JdbcRepository
+import io.micronaut.data.model.query.builder.sql.Dialect.POSTGRES
+import io.micronaut.data.repository.PageableRepository
+
+@JdbcRepository(dialect = POSTGRES) // <1>
+interface ToDoRepository : PageableRepository<ToDo?, Long?> // <2>
```

Comment:
should this be `<Todo,Long>` instead of `<Todo?,Long?>`?

Reply:
Done 👍